### PR TITLE
Release 0.17.5

### PR DESCRIPTION
**Pull request by euri10**
draft here : https://github.com/encode/uvicorn/releases/edit/untagged-f77adff66ddf2071a641

### CODE REVIEW

**Review by lovelydinosaur — APPROVED** [review]

**Comment by lovelydinosaur** [comment]
Oh, actually one last thing...
Perhaps worth considering this for the release, too?... https://github.com/encode/uvicorn/pull/1329

**Comment by lovelydinosaur** [comment]
I've bumped #1329 into `master` now too - could you make sure to update the release to include that, also.

**Comment by euri10** [comment]
updated draft: https://github.com/encode/uvicorn/releases/edit/untagged-011566e5082161fae795

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+### 0.17.5 - 2022-02-16
+
+### Fixed
+
+- Fix case where url is fragmented in httptools protocol (#1263) 2/16/22
+- Fix WSGI middleware not to explode quadratically in the case of a larger body (#1329) 2/16/16
```
Comment (vytas7) [inline CHANGELOG.md:8]: Thanks for merging #1329!
But hey... while I did start working on it a while ago, that wasn't 5 years ago :thinking:

Comment (euri10) [reply CHANGELOG.md:8]: /hide
I wasn't born at that time :turtle: !

Comment (lovelydinosaur) [reply CHANGELOG.md:8]: Just since we're here - I don't think we really need dates against individual PRs.
The release date is what matters.

Comment (euri10) [reply CHANGELOG.md:8]: agreed!

**Pull request merged**